README.rst

```diff
@@ -45,7 +45,7 @@ With a few steps, you can start in your client repository using CI confiurations
   script:
     - .ci_config/travis.sh
 
-* Note that `.ci_config` is the REQUIRED name of the cloned folder; it is hardcoded on `industrial_ci` side so you must adhere to it.
+* Note: The name `.ci_config` is NO longer REQUIRED for the cloned folder since version 0.3.2; you can name it whatever you like.
```

Comment:
> -> "you can name it whatever you like, a hidden folder is recommended"

@ipa-mdl Actually, is there a reason for this system-wise?
I've chosen to use hidden folder as a cleaner solution, even though CI scripts only run temporarily during the CI jobs. But other than that I don't see a reason.